tasks/libs/releasing/documentation.py

```diff
@@ -21,12 +21,6 @@ def _stringify_config(config_dict):
     return {key: str(value) for key, value in config_dict.items()}
 
 
-def nightly_entry_for(agent_major_version):
-    if agent_major_version == 6:
-        return "nightly"
-    return f"nightly-a{agent_major_version}"
-
-
 def release_entry_for(agent_major_version):
     return f"release-a{agent_major_version}"
```

Comment:
This looks like it will return an incorrect section name now, but I wonder if it's still used at all

Reply:
Indeed, you're right, I fixed it now. The tests weren't catching this one since they use this same function to create the entries for test inputs.